content/zh-cn/blog/_posts/2022-04-29-kubernetes-1.23-release-interview.md

```diff
@@ -0,0 +1,806 @@
+
+REY LEJANO: Moving to three releases a year from four, in my opinion, has been an improvement, because we support the last three releases, and now we can actually support the last releases in a calendar year instead of having 9 months out of 12 months of the year.
+-->
+**CRAIG BOX：你如何看待每年发布三次而不是四次的日程安排的变化？**
+
+REY LEJANO：在我看来，从一年四个版本转为三个版本，是一种进步，因为我们支持最后三个版本，
+现在我们实际上可以支持在一个日历年内的最后一个版本，而不是在 12 个月中只有 9 个月。
+
+<!--
+**CRAIG BOX: The next event on the calendar is a [Kubernetes contributor celebration](https://www.kubernetes.dev/events/kcc2021/) starting next Monday. What can we expect from that event?**
+
+REY LEJANO: This is our second time running this virtual event. It's a virtual celebration to recognize the whole community and all of our accomplishments of the year, and also contributors. There's a number of events during this week of celebration. It starts the week of December 13.
+-->
+**CRAIG BOX：日历上的下一个活动是下周一开始的 [Kubernetes 贡献者庆祝活动](https://www.kubernetes.dev/events/kcc2021/)。我们可以从该活动中期待什么？**
+
+REY LEJANO：这是我们第二次举办这个虚拟活动。这是一个虚拟的庆祝活动，以表彰整个社区和我们今年的所有成就，以及贡献者。
+在这一周的庆祝活动中，有许多活动。它从 12 月 13 日那一周开始。
+
+<!--
+There's events like the Kubernetes Contributor Awards, where SIGs honor and recognize the hard work of the community and contributors. There's also a DevOps party game as well. There is a cloud native bake-off. I do highly suggest people to go to [kubernetes.dev/celebration](https://www.kubernetes.dev/events/past-events/2021/kcc2021/) to learn more.
+-->
+有像 Kubernetes 贡献者奖这样的活动，SIG 对社区和贡献者的辛勤工作进行表彰和奖励。
+也有一个 DevOps 聚会游戏。还有一个云原生的烘烤活动。我强烈建议人们去
+[kubernetes.dev/celebration](https://www.kubernetes.dev/events/past-events/2021/kcc2021/)
+了解更多。
+
+<!--
+**CRAIG BOX: How exactly does one judge a virtual bake-off?**
+
+REY LEJANO: That I don't know. [CHUCKLES]
+-->
+**CRAIG BOX： 究竟如何评判一个虚拟的烘焙比赛呢？**
+
+REY LEJANO：那我不知道。[笑]
+
+<!--
+**CRAIG BOX: I tasted my scones. I think they're the best. I rate them 10 out of 10.**
+
+REY LEJANO: Yeah. That is very difficult to do virtually. I would have to say, probably what the dish is, how closely it is tied with Kubernetes or open source or to CNCF. There's a few judges. I know Josh Berkus and Rin Oliver are a few of the judges running the bake-off.
+-->
+**CRAIG BOX：我尝了尝我的烤饼。我认为他们是最好的。我给他们打了 10 分（满分 10 分）。**
+
+REY LEJANO：是的。这是很难做到的。我不得不说，这道菜可能是什么，它与 Kubernetes 或开源或与 CNCF 的关系有多密切。
+有几个评委。我知道 Josh Berkus 和 Rin Oliver 是主持烘焙比赛的几个评委。
+
+<!--
+**CRAIG BOX: Yes. We spoke with Josh about his love of the kitchen, and so he seems like a perfect fit for that role.**
+
+REY LEJANO: He is.
+-->
+**CRAIG BOX：是的。我们与 Josh 谈到了他对厨房的热爱，因此他似乎非常适合这个角色。**
+
+REY LEJANO：他是。
+
+<!--
+**CRAIG BOX: Finally, your wife and yourself are expecting your first child in January. Have you had a production readiness review for that?**
+
+REY LEJANO: I think we failed that review. [CHUCKLES]
+-->
+**CRAIG BOX：最后，你的妻子和你自己将在一月份迎来你们的第一个孩子。你是否为此进行过生产准备审查？**
+
+REY LEJANO：我认为我们没有通过审查。[笑]
```

Comment:
This format is weird in Mandarin.

Reply:
Please help to give specific instructions

Reply:
> This format is weird in Mandarin.

Please help to give specific instructions

Reply:
I would prefer this format.
```suggestion
REY LEJANO：我认为我们没有通过审查。（笑）
```

How do you think? It's fine for me if you think the original version is better.